Calculator/ViewModel/CalculatorViewModel.swift

```diff
@@ -0,0 +1,95 @@
+import Foundation
+
+struct CalculatorViewModel : CalculatorOperationsProtocol {
```

Comment:
given all the functions here are mutating, perhaps a class could be better than a struct?